elfeed-search.el

```diff
@@ -699,22 +703,52 @@ expression, matching against entry link, title, and feed title."
                              line))
          (move-to-column column)))))
 
-(defun elfeed-search-update (&optional force)
+(defun elfeed-entries-compare (entry-a entry-b)
```

Comment:
Please reorder these functions so the diff can be more cleanly read (see how it shows the `elfeed-search-update` function's first line being replaced, but the new line is far below it in the diff).